packages/@o3r/eslint-config-otter/rules/typescript/eslint-typescript.cjs

```diff
@@ -53,7 +53,8 @@ module.exports = {
       {
         'accessibility': 'explicit',
         'overrides': {
-          'constructors': 'off'
+          'constructors': 'off',
+          'properties': 'off'
```

Comment:
Are you sure we want to deactivate it ?

Reply:
It's normal that we have stricter rules than Angular 
I would fix it with a regexp to add the `public` and keep our recommendation

Reply:
I am jumping into the topic as I received multiple feedbacks that angular is too verbose, and we are adding extra boilerplate on top of generated angular code. 
IMO if the 'public' accessor is not present, we should not raise an 'error'; The 'public' is implicit.